cost-analyzer/templates/query-service-deployment-template.yaml

```diff
@@ -0,0 +1,113 @@
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: query-service
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app: query-service
+  serviceName: {{ .Release.Name }}-query-service-load-balancer
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: query-service
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app: query-service
+    spec:
+      restartPolicy: Always
+      serviceAccountName: {{ template "query-service.serviceAccountName" . }}
+      volumes:
+        {{- $etlBackupBucketSecret := "" }}
+        {{- if .Values.kubecostModel.etlBucketConfigSecret }}
+            {{- $etlBackupBucketSecret = .Values.kubecostModel.etlBucketConfigSecret }}
+        {{- else if and .Values.global.thanos.enabled (ne (typeOf .Values.kubecostModel.etlBucketConfigSecret) "string") }}
+            {{- $etlBackupBucketSecret = .Values.thanos.storeSecretName }}
+        {{- end }}
+        {{- if $etlBackupBucketSecret }}
+        - name: etl-bucket-config
+          secret:
+           defaultMode: 420
+           secretName: {{ $etlBackupBucketSecret }}
+        {{- end }}
+      initContainers:
+        - name: config-db-perms-fix
+          image: busybox
+          command: ["sh", "-c", "/bin/chmod -R 777 /var/configs && /bin/chmod -R 777 /var/db"]
+          volumeMounts:
+            - name: persistent-configs
+              mountPath: /var/configs
+            - name: persistent-db
+              mountPath: /var/db
+          securityContext:
+            runAsUser: 0
+      containers:
+        - name: query-service
+          image: {{ .Values.kubecostModel.fullImageName }}
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: 9003
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            failureThreshold: 200
+          imagePullPolicy: Always
+          args: ["query-service"]
+          ports:
+            - name: tcp-model
+              containerPort: 9003
+              protocol: TCP
+            - name: tcp-frontend
+              containerPort: 9090
+              protocol: TCP
+          volumeMounts:
+            - name: persistent-configs
+              mountPath: /var/configs
+            - name: etl-bucket-config
+              mountPath: /var/configs/etl
+              readOnly: true
+            - name: persistent-db
+              mountPath: /var/db
+          env:
+            - name: CONFIG_PATH
+              value: /var/configs/
+            - name: DB_PATH
+              value: /var/db/
+            - name: ETL_FILE_STORE_ENABLED
+              value: "true"
+            - name: ETL_TO_DISK_ENABLED
+              value: "false"
+            {{- if $etlBackupBucketSecret }}
+            - name: ETL_BUCKET_CONFIG
+              value: "/var/configs/etl/object-store.yaml"
+            {{- end }}
+            - name: ETL_TO_DISK_ENABLED
+              value: "true"
```

Comment:
Looks like we have `ETL_TO_DISK_ENABLED` twice with conflicting definitions. Which should it be?

Reply:
I'm glad you're looking at this Michael, I don't know how I missed this. It should be true.